src/triage/cli.py

```diff
@@ -224,9 +224,30 @@ def __init__(self, parser):
             default='audition_config.yaml',
             help="config file for audition",
         )
+        parser.add_argument(
+            '-v', '--validate',
+            action='store_true',
+            help='validate before running audition'
+        )
+        parser.add_argument(
+            '--no-validate',
+            action='store_false',
+            dest='validate',
+            help="run audition without validation"
+        )
+        parser.add_argument(
+            '--validate-only',
+            action='store_true',
+            help="only validate the config file not running audition"
+        )
+        parser.add_argument(
+            '-d', '--directory',
+            help='directory to store the result plots from audition',
+        )
         parser.set_defaults(
             directory=None,
             validate=True,
+            validate_only=False,
```

Comment:
yep, it looks like the reason it broke was just that this default hadn't been set here.

note, however, that now that you've squashed things into a single command, (rather than a tiered one requiring defaults to be set on the base command), this `set_defaults` call is redundant and unDRY. These defaults can be moved to their respective `add_argument` calls, as necessary.